pylint/checkers/base/basic_checker.py

```diff
@@ -767,6 +773,10 @@ def visit_tryfinally(self, node: nodes.TryFinally) -> None:
         assert self._tryfinallys is not None
         self._tryfinallys.append(node)
 
+        for final_node in node.finalbody:
+            for return_node in final_node.nodes_of_class(nodes.Return):
+                self.add_message("return-in-finally", node=return_node)
```

Comment:
We add confidence to new messages
```suggestion
                self.add_message("return-in-finally", node=return_node, confidence=HIGH)
```

Reply:
Thanks! Done.